.github/CODEOWNERS

```diff
@@ -1,2 +1,4 @@
+* @dbt-labs/core-group
```

Comment:
I noticed both I and Adrian were added automatically as reviewers on this PR. We're both on Adapters, which is a sub-team of `core-group`. I've never interacted with this repo. Is it possible this should be `core-team`?

Reply:
let me update to core-team... i wasn't sure the difference between the two.